hybris/vulkan/vulkan.c

```diff
@@ -104,6 +105,41 @@ VkResult vkEnumerateInstanceExtensionProperties(const char* pLayerName, uint32_t
     return ws_vkEnumerateInstanceExtensionProperties(pLayerName, pPropertyCount, pProperties);
 }
 
+VkResult vkGetPhysicalDeviceSurfaceCapabilitiesKHR(VkPhysicalDevice physicalDevice, VkSurfaceKHR surface, VkSurfaceCapabilitiesKHR* pSurfaceCapabilities)
+{
+    VkResult ret;
+    if (_vkGetPhysicalDeviceSurfaceCapabilitiesKHR == NULL) {
+        _vkGetPhysicalDeviceSurfaceCapabilitiesKHR = (VkResult (*)(VkPhysicalDevice, VkSurfaceKHR, VkSurfaceCapabilitiesKHR*))
+            (*_vkGetInstanceProcAddr)((VkInstance)physicalDevice, "vkGetPhysicalDeviceSurfaceCapabilitiesKHR");
+    }
+
+    ret = (*_vkGetPhysicalDeviceSurfaceCapabilitiesKHR)(physicalDevice, surface, pSurfaceCapabilities);
+    // Surface size will be determined by the extent of a swapchain targeting the surface
+    pSurfaceCapabilities->currentExtent = (VkExtent2D){0xFFFFFFFF, 0xFFFFFFFF};
```

Comment:
this looks a bit weird, can you make the comment a bit more explanatory?

Reply:
That was an old remnant from some early testing, the code was not needed based on more testing.